app/src/main/java/ie/macinnes/tvheadend/BootCompletedReceiver.java

```diff
@@ -0,0 +1,16 @@
+package ie.macinnes.tvheadend.service;
```

Comment:
The build is failing, because the class can't be found due to the package being incorrect :) Drop the .service and this should work!